awx/main/tasks/jobs.py

```diff
@@ -176,9 +186,19 @@ def build_execution_environment_params(self, instance, private_data_dir):
         if settings.AWX_ISOLATION_SHOW_PATHS:
             params['container_volume_mounts'] = []
             for this_path in settings.AWX_ISOLATION_SHOW_PATHS:
-                # Using z allows the dir to mounted by multiple containers
-                # Uppercase Z restricts access (in weird ways) to 1 container at a time
-                params['container_volume_mounts'].append(f'{this_path}:{this_path}:z')
+                # Verify if a mount path and SELinux context has been passed
+                if this_path.count(':') == 2:
+                    src, dest, scontext = this_path.split(':')
+                    if scontext not in _ALLOWED_EE_VOLUMES_MOUNT_TYPES:
+                        scontext = 'z'
+                    params['container_volume_mounts'].append(f'{src}:{dest}:{scontext}')
+                elif this_path.count(':') == 1:
+                    src, dest = this_path.split(':')
+                    params['container_volume_mounts'].append(f'{src}:{dest}:z')
+                else:
+                    # Using z allows the dir to mounted by multiple containers
```

Comment:
nitpicky comment -> "Using z allows the dir to be mounted by multiple containers"